src/app/components/profile/profile.component.html

```diff
@@ -1,101 +1,117 @@
-		</div>
-		<div class="user-details">
-			<div class="item">
-				<span class="text-light-black fw-semibold fs-12">
-					First Name<br>
-				</span>
-				<div class="content fw-regular">
-					{{user.first_name}}
-				</div>
-			</div>
-			<div class="item">
-				<span class="text-light-black fw-semibold fs-12">
-					Last Name<br>
-				</span>
-				<div class="content fw-regular">
-					{{user.last_name}}
-				</div>
-			</div>
-			<div class="item">
-				<span class="text-light-black fw-semibold fs-12">
-					Affiliated to<br>
-				</span>
-				<div class="content fw-regular">
-					{{user.affiliation}}
-				</div>
-			</div>
-		</div>
-		<div class="item">
-			<div class="highlight">
-				<span class="fw-regular fs-14">Profile Completed</span>
-				<br> <span class="text-med-black">{{pcomp}}</span>
-				<div>
-				</div>
-				<br>
-				<div class="item">
-					<button class="btn btn-waves-effect waves-dark grad-btn grad-btn-dark
-						fs-16 rm-pad-top" (click)="isTokenModalVisible= true;">Get your Auth Token</button>
-				</div>
-				<div class="item change-password">
-					<a (click)="updatePassword()" class="dark-link pointer">
-						<strong class="fw-semibold fs-15 change-password-text">Change Password</strong>
-					</a>
-					<button (click)="updateUserDetails()" class="btn btn-waves-effect
-						waves-dark grad-btn grad-btn-dark rm-pad-top fs-16">Update Profile</button>
-				</div>
-
-				<div class="modal-container" [class.hidden]="!isTokenModalVisible">
-					<div class="modal-card">
-						<div class="title">
-							API Authentication Token
-						</div>
-						<div class="form row">
-							<div class="col-lg-10 col-md-10 col-sm-10 col-xs-10">
-								<app-input [label]="'token'" [placeholder]="'Auth Token'"
-									[type]="'password'" [theme]="'dark'" [value]="token" [readonly]="true"
-									#formtoken></app-input>
+<div class="main_content">
+	<app-side-bar style="background-color: rgb(250,250,250)"></app-side-bar>
+	<div class="container-fluid main_content">
+		<div class="row">
+			<div class="dashboard">
+				<div class="md-container  main_content" id="card_container">
+					<div class="ev-md-container ev-card-panel ev-z-depth-5 challenge-title-container">
+						<div class="col s12 m3 l3 center"> 
+							<img class="profile-photo" src="assets/images/profile.png" />
+							<div class="item">
+								<div class="content fw-regular">
+									{{user.username}}
+								</div>
+								<div class="fs-15">
+									{{user.email}}
+								</div>
+							</div>
+							<div class="item" id="profile_layout">
+								<div class="highlight">
+									<span class="fw-regular fs-14">Profile Completed</span>
+									<br> <span class="text-med-black">{{pcomp}}</span>
+								</div>
+							</div>
+							<br>
+							<br>
+							<div class="item">
+								<button class="btn btn-waves-effect waves-dark grad-btn grad-btn-dark fs-16 rm-pad-top" (click)="isTokenModalVisible= true;"> Get your Auth Token</button>
```

Comment:
Please format and make it max-line length to 80.

Reply:
Here, I mean to make the line-length to <=80. For line number 28 it is going to 187. Please let me know if you didn't get this.